### ISSUE TITLE
Node.js교과서 16장 CSP

### ISSUE THREAD

**Issue by Singularity2050** [ZeroCho/nodejs-book#118]
안녕하세요 16장에 구글 storage와 연결해서 사용할려고하는데 Content Security Policy 때문에 미리보기가 안되네요 ㅠㅠ 어떻게해야하죠?
![스크린샷 2020-09-04 오전 11 15 11](https://user-images.githubusercontent.com/67400401/92191950-e9539080-ee9f-11ea-9200-d5499b43b6a4.png)

`<meta http-equiv="Content-Security-Policy" content="img-src https://storage.googleapis.com 'self' data: https:"> `
메타테그를 이렇게 넣는데되 안되네요 ㅠㅠ 도와주세요..

**Comment by ZeroCho** [comment]
helmet이 4.0으로 업데이트돼서 그런 것 같습니다
```js
app.use(helmet())
```
을
```js
app.use(helmet({ contentSecurityPolicy: false }))
```
로 helmet 코드를 수정해주세요.

**Comment by Singularity2050** [comment]
감사합니다 아 그리고
![스크린샷 2020-09-04 오후 6 53 45](https://user-images.githubusercontent.com/67400401/92226662-0a3bd600-eee0-11ea-95dc-8ae14c7ffc49.png)
16장에 Google Cloud Function 만드는데 계속 function.js  doesn't exist뜨는데 어떻게 하죠? ㅠㅠ

**Comment by Singularity2050** [comment]
아니요~!  그 선생님이 깃하즈에있는 16.5에 gcp-upload에 index.js package.json파일을 npm i 해서
zip으로 압축해서 올렸는습니다 ! 노드버전은 12버전으로 했고요 resizeAndUpload 함수로 등록했습니다

2020년 9월 5일 (토) 오후 3:16, Hyunyoung Cho <notifications@github.com>님이 작성:

**Comment by ZeroCho** [comment]
혹시 cloud functions가 아니라 firebase functions에 잘못 올리신 적이 있으신가요?
기존 functions를 지우고 다시 함수를 생성하고 zip을 업로드해보세요.

**Comment by ZeroCho** [comment]
helmet 5버전에서는 다음과 같이 해야 외부 이미지, 아이프레임 등이 제대로 표시됩니다.
app.use(
  helmet({ contentSecurityPolicy: false, crossOriginEmbedderPolicy: false, crossOriginResourcePolicy: false }),
);
